src/test/java/TestWriter.java

```diff
@@ -0,0 +1,25 @@
+import catalin.Matrix;
+import catalin.Reader;
+import catalin.Writer;
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+
+public class TestWriter {
+
```

Comment:
Here you can use something like     
@Rule
public TemporaryFolder folder = new TemporaryFolder();
and inside of this you will create a file. This is good in  order to not have a  file after the test is finished.